### ISSUE TITLE
Design microcopy solution for "null" state of FEC filing ID

### ISSUE THREAD

**Issue by jenniferthibault** [18F/openFEC-web-app#1841]
From the Filings data table, https://fec-stage-proxy.18f.gov/data/filings/?data_type=processed
We rarely expose "null" states verbatim, and usually offer explanation for a missing piece of data. To help users understand what "null" means here, offer a plain language microcopy fill in.

<img width="968" alt="screen shot 2017-02-16 at 9 42 42 am" src="https://cloud.githubusercontent.com/assets/11636908/23025688/c77d98a0-f42c-11e6-8ca7-bfd15c1cd9dd.png">
<img width="967" alt="screen shot 2017-02-16 at 9 42 01 am" src="https://cloud.githubusercontent.com/assets/11636908/23025692/cb2af9f2-f42c-11e6-8923-77f6282f57d3.png">

**Steps to completion**
- Microcopy designed
- Copy implemented in table

(This issue should not be a barrier to the first release)

**Comment by jenniferthibault** [comment]
When I started looking into this with @jontours and @LindsayYoung , we realized that the underlying factors to this "null" state pointed to some improvements to the table logic that need to be resolved or reconciled before this really becomes a content design task.

In this issue, the Version/amendment status and the FEC-ID number should be able to inform one another, so it's especially odd when we have a "Version unknown" status paired with an Amendment status AND an FEC ID number, like the following:

<img width="907" alt="screen shot 2017-04-28 at 11 11 04 am" src="https://cloud.githubusercontent.com/assets/11636908/25541413/6ed32346-2c03-11e7-98c7-51495619e5e7.png">


This scenario of mixed states is one we see often for Form 1 (Statements of organization) & Form 2 (Statements of candidacy) filings: https://beta.fec.gov/data/filings/?data_type=processed&form_type=F1&form_type=F2

<img width="978" alt="screen shot 2017-04-28 at 11 08 24 am" src="https://cloud.githubusercontent.com/assets/11636908/25541316/08651b6e-2c03-11e7-9d3a-32470b19f179.png">
 
**Data logic tasks:**
- See if we can determine from the data (and surface) the correct Version of the document and amendment status
- See if there are any remaining instances of missing FEC ID numbers, when then occur, and why

Then, when we know those answers it becomes a micro-copy design question. 

I'm moving this out of "in progress" and back to "ready" so that next week we can discuss with @noahmanger whether its still something we want to take on this sprint. 

@LindsayYoung,  @jontours, anything you'd add or correct in what I've recapped?

**Comment by jontours** [comment]
That's a good sum-up of what we talked about.  After looking into the data a bit I think I've found the culprit for the case of when the fec-id is not null, yet the version is showing as unknown.  I think it will be easier to fix via a schema update.   I'll open a PR for that.  As of right now that would fix around 100000 records.

Now, 100k may not seem like a lot when there are around 3 million filings, but I think many of them occur later in chronological time, and hence show up more than if distributed equally across time.
